modules/shared/src/main/scala/io/constellationnetwork/currency/dataApplication/package.scala

```diff
@@ -524,22 +658,24 @@ object dataApplication {
 
   case class DataApplicationBlock(
     roundId: RoundId,
-    updates: NonEmptyList[Signed[DataUpdate]],
-    updatesHashes: NonEmptyList[Hash]
-  ) extends Encodable[NonEmptyList[Hash]] {
-    override def toEncode: NonEmptyList[Hash] = updatesHashes
-    override def jsonEncoder: Encoder[NonEmptyList[Hash]] = implicitly
+    updates: NonEmptyList[DataTransactions],
+    updatesHashes: NonEmptyList[NonEmptyList[Hash]]
```

Comment:
I think it's not `updates` anymore but `transactions`, and `transactionsHashes` 🤔

Reply:
👍🏼